### PR TITLE
[HOPSWORKS-2131] Append sql migration to new python_environment table

### PR DESCRIPTION
**Pull request by robzor92**

### CODE REVIEW

File: files/default/sql/ddl/updates/2.1.0.sql [hunk 1]
```diff
@@ -163,6 +163,18 @@ CREATE TABLE IF NOT EXISTS `python_environment` (
   CONSTRAINT `FK_PYTHONENV_PROJECT` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 
+INSERT INTO `hopsworks`.`python_environment` (`project_id`, `python_version`)
+SELECT `id`, `python_version`
+FROM `hopsworks`.`project`
+WHERE `python_version` IS NOT NULL
+AND `conda` = true;
+
+SET SQL_SAFE_UPDATES = 0;
```
Comment (tkakantousis) [inline files/default/sql/ddl/updates/2.1.0.sql:172]: I thought `SQL_SAFE_UPDATES` is used for update queries without a `where` clause. Did you try it without turning SQL_SAFE_UPDATES off?

**Review by tkakantousis — APPROVED** [review]

File: files/default/sql/ddl/updates/2.1.0.sql (hunk 1 above)
Comment (robzor92) [reply files/default/sql/ddl/updates/2.1.0.sql:172]: It doesn't work as the UPDATE itself does not have a WHERE clause. The WHERE clause you see there is in the inner SELECT so I need to disable the `SQL_SAFE_UPDATES` to run the UPDATE statement.

Comment (robzor92) [reply files/default/sql/ddl/updates/2.1.0.sql:172]: I've run the migration and rollback and couple of times now with projects that have python enabled and disabled, and redeployed hopsworks to see if it works. This code is now good to merge.

Comment (tkakantousis) [reply files/default/sql/ddl/updates/2.1.0.sql:172]: oh yes you are correct

**Pull request merged**